src/client/java/teammates/client/scripts/RepairFeedbackSessionNameWithExtraWhiteSpace.java

```diff
@@ -0,0 +1,228 @@
+package teammates.client.scripts;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.jdo.PersistenceManager;
+import javax.jdo.Query;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.common.datatransfer.FeedbackSessionAttributes;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.StringHelper;
+import teammates.storage.api.FeedbackSessionsDb;
+import teammates.storage.datastore.Datastore;
+import teammates.storage.entity.FeedbackQuestion;
+import teammates.storage.entity.FeedbackResponse;
+import teammates.storage.entity.FeedbackResponseComment;
+import teammates.storage.entity.FeedbackSession;
+
+/**
+ * Previews and removes extra spaces in the feedbackSessionName field of
+ * FeedbackSessionAttributes, FeedbackResponseAttributes,
+ * FeedbackQuestionAttributes and FeedbackResponseCommentAttribute.
+ */
+public class RepairFeedbackSessionNameWithExtraWhiteSpace extends RemoteApiClient {
+    private static final boolean isPreview = true;
+    
+    private FeedbackSessionsDb feedbackSessionsDb = new FeedbackSessionsDb();
+    
+    private List<String> courseIdsToRunOn = Arrays.asList();
```

Comment:
Technically, there is still a possibility that a multi-space issue was created after I ran the preview script. So  I guess I should run it as before, but preview set to false.